py4j-web/contributing.rst

```diff
@@ -113,7 +113,6 @@ suite across the supported python versions.
   ./gradlew assemble
 
   # install test requirements
```

Comment:
and I would add some comments to guide users to come back to the Py4J home directory, or add `cd ..`

Reply:
Sounds good. Let me address the comments, thanks!